include/swift/Parse/SyntaxParsingCache.h

```diff
@@ -86,16 +80,15 @@ class SyntaxParsingCache {
   /// reused for a new syntax tree.
   llvm::Optional<Syntax> lookUp(size_t NewPosition, SyntaxKind Kind);
 
-  /// Turn recording of reused ranges on
-  void setRecordReuseInformation() { RecordReuseInformation = true; }
-
-  /// Return the ranges of the new source file that have been successfully
-  /// looked up in this cache as a (start, end) pair of byte offsets in the
-  /// post-edit file.
-  std::vector<SyntaxReuseRegion> getReusedRanges() const {
-    return ReusedRanges;
+  const std::unordered_set<unsigned> &getReusedNodeIds() const {
```

Comment:
It would be good to have a type alias of unsigned as the ID type, so we can search more easily.